app/javascript/mastodon/features/compose/components/compose_form.js

```diff
@@ -21,6 +21,7 @@ import ImmutablePureComponent from 'react-immutable-pure-component';
 import { length } from 'stringz';
 import { countableText } from '../util/counter';
 import Icon from 'mastodon/components/icon';
+import { publish_button_text } from 'mastodon/initial_state';
```

Comment:
If you were trying to avoid conflict by using snake_case, you can just use alias, e.g. `import { publishButtonText as $publishButtonText } …`.